lib/cloud_controller/deployment_updater/calculators/all_instances_routable.rb

```diff
@@ -0,0 +1,26 @@
+module VCAP::CloudController
+  module DeploymentUpdater
+    module Calculators
+      class AllInstancesRoutable
```

Comment:
**nit** I wonder if this could be less specific, something like `AllInstancesReady` or `AllInstancesRunning` ?